clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/AbstractElasticsearchDAOSpec.scala

```diff
@@ -35,17 +33,15 @@ abstract class AbstractElasticsearchDAOSpec(actorSystemName: String)
 
   protected def initialize(): Future[Assertion] = {
     // Set *all* existing index replicas to 0, including elasticsearch's internal indexes
-    val entity = new NStringEntity(
+    val entity = HttpEntity(
       """|{
          |  "index" : {
          |    "number_of_replicas" : 0
          |  }
-         |}
-         |""".stripMargin,
-      ContentType.APPLICATION_JSON
```

Comment:
Is content type no longer required?

Reply:
It now defaults to `"application/json; charset=utf-8"` in `HttpEntity.apply`